python/paddle/fluid/imperative/layers.py

```diff
@@ -53,6 +70,44 @@ def full_name(self):
         """
         return self._full_name
 
+    @property
+    def param_attr(self):
+        return self._helper.param_attr
+
+    @property
+    def bias_attr(self):
+        return self._helper.bias_attr
+
+    def create_parameter(self,
+                         attr,
+                         shape,
+                         dtype,
+                         is_bias=False,
+                         default_initializer=None):
```

Comment:
pass param attr here?

Reply:
user can pass param attr either from here and from the constructor of the layer_helper